content/gu/docs/tutorials/kubernetes-basics/update/update-intro.html

```diff
@@ -0,0 +1,137 @@
+---
+title: रोलिंग अपडेट करना 
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="hi">
```

Comment:
_Hindi?_

Reply:
Yes it's localized in hindi.

Reply:
Shouldn't it be Gujurati? That's what I'd expect to see.

Reply:
Oh..okay okay..
I misunderstood.
Let me localized it in Gujarati.